java/client/src/main/java/glide/api/models/commands/scan/ScanOptions.java

```diff
@@ -0,0 +1,59 @@
+/** Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands.scan;
```

Comment:
make sure to add scan package to [module-info.java](https://github.com/aws/glide-for-redis/blob/main/java/client/src/main/java/module-info.java)